src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
```

Comment:
The method 'doWork' should not return a string. Instead, it should print an informative message to the console. Also, make sure to specify the type of machine, in this case, 'Truck'.